benchmark_test.go

```diff
@@ -0,0 +1,80 @@
+package waifu2x
+
+import (
+	"image"
+	"image/png"
+	"os"
+	"testing"
+)
+
+func Benchmark01(b *testing.B) {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 14 of function Benchmark01() is high (> 10)